device_calendar/example/lib/presentation/pages/calendar_event.dart

```diff
@@ -688,6 +711,45 @@ class _CalendarEventPageState extends State<CalendarEventPage> {
     }
   }
 
+  void _updateDaysOfWeek(DayOfWeekGroup selectedGroup, bool isSelected) {
+    var days = selectedGroup.getDays;
+
+    if (isSelected) {
+      _daysOfWeek.addAll(days.where((a) => _daysOfWeek.every((b) => a != b)));
```

Comment:
what happens if _daysOfWeek already has values that are not in days - do we need to remove them?

Reply:
Nothing, that `allAll` condition will only add whatever it's not in `_daysOfWeek` list.
i.e. If `_daysOfWeek` already have Monday, Tuesday and Wednesday, choosing `DayOfWeekGroup.Weekday` will only add Thursday and Friday.

Reply:
but what happens if _daysOfWeek has sat or sun... shouldn't they be removed if Weekday is specified?

Reply:
ok how about adding `None` and `Custom` as you've mentioned before and have `DayOfTheWeekGroup` as radio buttons in the example app?

This way if `_daysOfWeek` has Monday to Saturday, `Custom` will be chosen, choosing `Weekday` will just remove Saturday.

Reply:
@nickrandolph I've changed it to be a radio button with `None` and `Custom` added to the enum